config/pepperConfig.js.ctmpl

```diff
@@ -332,3 +332,24 @@ var DDP_ENV = {
         DDP_ENV['projectGAToken'] = "TBD";
     {{end}}
 {{end}}
+
+{{if eq $study_guid "brugada"}}
```

Comment:
If we need GoogleAnalytics  for Brugada can we please add GA token for Brugada prod